### DISCUSSION TITLE
Cache in RAM?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#5685] [Q&A] [answered]
Is it possible? 

In Firefox I can change a few values in `about:config` (like disabling `browser.cache.disk.enable`, and something else, I forget) to achieve this. I didn't find a setting for this in `qute://settings/`, but I may have looked in the wrong place?

**Comment by mschilli87** [comment]
@basedtho: Since qutebrower uses webengine, knowing the corresponding Chromium-option(s) would be more helpful than Firefox's.

**Comment by ghost** [comment, marked as answer]
okay, so setting content.cache.size to 1 seems to solve this

**Reply by toofar** [reply]
That is good to know, although presumably there is a difference between caching only in ram and not caching at all. You could also move the webengine cache folder (or the whole qute cache folder) to a ramdisk.
